client/src/features/PostCreator/PostCreator.tsx

```diff
@@ -0,0 +1,48 @@
+import './PostCreator.css'
+import { useState } from 'react'
+import { useNavigate } from 'react-router-dom'
+
+export default function PostCreator() {
+  const [title, setTitle] = useState('');
+  const [body, setBody] = useState('');
+  const [tags, setTags] = useState('');
+  
+  const navigate = useNavigate();
+
+  function submitPost() {
```

Comment:
This could be my own preference, so take this with a grain of salt - I always see local functions defined as lambdas, so `const submitPost = () => { ... }`. This isn't wrong, but it looks a bit odd to me? Maybe others disagree though.

Reply:
went ahead and changed it for consistency if nothing else!